content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -11,9 +11,72 @@ This document describes the Prisma CLI commands, arguments, and options.
 
 </TopBlock>
 
+## Installation
+
+The Prisma CLI is typically installed as a **development dependency**, that's why the `--save-dev` (npm) and `--dev` (Yarn) options are used in the commands below.
+
+<Admonition>
+
+We **recommend that you install the Prisma CLI locally** in your project's `package.json` to avoid version conflicts that can happen with a global installation.
```

Comment:
```suggestion
We **recommend that you install the Prisma CLI locally** (without `-g`) in your project's `package.json` to avoid version conflicts that can happen with a global installation.
```